Formula/gotify-cli.rb

```diff
@@ -0,0 +1,25 @@
+class GotifyCli < Formula
+  desc "Command-line interface for pushing messages to gotify/server"
+  homepage "https://github.com/gotify/cli"
+  url "https://github.com/gotify/cli/archive/refs/tags/v2.2.0.tar.gz"
+  sha256 "d4e6fbc087c6cc4b320b3fcae4c28d65d514b7e55c95e4c0e764abb3a6c94064"
+  license "MIT"
+  head "https://github.com/gotify/cli.git", branch: "master"
+
+  livecheck do
+    url "https://github.com/gotify/cli/releases/latest"
+    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-X main.Version=#{version}
+                                       -X main.BuildDate=2021-12-09-12:45:29
```

Comment:
I was not sure how to insert this dynamically, can someone help me with this?

Reply:
```suggestion
                                       -X main.BuildDate=#{time.iso8601}
```

Reply:
This will produce the date in a slightly different format: `2021-12-09T08:28:51Z`

If you want the format you've specified in particular:
```suggestion
                                       -X main.BuildDate=#{time.strftime("%F-%R:%S")}
```
We should probably go with a standardised date format (e.g. `iso8601`) unless there's a good reason not to, however.

Reply:
Thank you! I think it's ok to go with `time.iso8601`, since it's only used in the version output.